pkg/proxy/kafka.go

```diff
@@ -340,23 +341,49 @@ func handleConnection(pair *connectionPair, c *proxyConnection,
 	}
 }
 
+func handleResponse(pair *connectionPair, c *proxyConnection,
+	record *kafkaLogRecord, handler kafkaRespMessageHander) {
+	for {
+		rsp, err := kafka.ReadResponse(c.conn)
+		if err != nil {
+			if err == io.EOF || err == io.ErrUnexpectedEOF {
+				c.Close()
+				return
+			}
+
+			if record != nil {
+				record.log(accesslog.TypeResponse, accesslog.VerdictError,
+					kafka.ErrInvalidMessage,
+					fmt.Sprintf("Unable to parse Kafka response: %s", err))
+			}
+
+			log.WithError(err).Error("Unable to parse Kafka response")
+			continue
```

Comment:
No need for this `continue`

Reply:
It's a copy paste from `handleRequest`, we should remove it from both places